client/stylesheets/_toCat.scss

```diff
@@ -4221,16 +4221,15 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileReceivedUl {
-    width: 55%;
-    margin-left: 21% !important;
-    margin-right: 21% !important;
+    width: 97%;
+    margin: 0;
 }
 
 .profileReceivedLi {
-    width: 100%;
+    height: 475px;
     margin-bottom: 20px;
     padding: 10px;
-    border: 2px solid #75D855;
+    border: 2px solid #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces